### PR TITLE
Viewbox

### PR DESCRIPTION
**Pull request by weijeffrey**
still stuck on viewbox but running out of time. 
Sorry

### CODE REVIEW

**Comment by h8rry** [comment]
No problem! Just should the middle with the right viewing port

File: js/application.js [hunk 1]
```diff
@@ -356,10 +359,13 @@ var graphTrail = function() {
...
   for (var j=1; j < game.monster.trail.length; j+=2){
     graphCoordArrayX.push([200+(game.monster.trail[j]*120)])
+    console.log(game.monster.trail[j]);
+    console.log(graphCoordArrayX)
   }
 
+
   for (var k=0; k < graphCoordArrayX.length; k++) {
-    graphCoordArrayXY.push( [graphCoordArrayX[k]- playScreen.getBoundingClientRect().left, graphCoordArrayY[k] - 195] )
+    graphCoordArrayXY.push( [graphCoordArrayX[k]- (playScreen.getBoundingClientRect().left), graphCoordArrayY[k] - 195] )
```
Comment (f3r) [inline js/application.js:368]: missing semicolon

File: js/application.js [hunk 2]
```diff
@@ -356,10 +359,13 @@ var graphTrail = function() {
 
   for (var j=1; j < game.monster.trail.length; j+=2){
     graphCoordArrayX.push([200+(game.monster.trail[j]*120)])
+    console.log(game.monster.trail[j]);
+    console.log(graphCoordArrayX)
```
Comment (f3r) [inline js/application.js:363]: missing semicolon

File: js/application.js [hunk 3]
```diff
@@ -449,7 +457,10 @@ $(document).ready(function() {
 });
   //player switch
   $(".pigReadyButton").mousedown(function(){
-    switchToPredator()
+    var oldSrc = "/assets/images/Piglet_KH.png"
```
Comment (f3r) [inline js/application.js:460]: missing semicolon

**Pull request merged**